index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <script src="https://kit.fontawesome.com/8b3d2372ca.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,500;0,700;1,400&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body>
+
+    <div class="container">
+        <div class="navbar">
+            <div class="logo">
+                G
+            </div>
+            <div class="menu">
+                <i class="fas fa-bars"></i>
+            </div>
+        </div>
```

Comment:
- [ ] The container of this section should be a `<header>` tag. Please kindly fix that as it is part of the project requirements. This also makes your html more semantic, which is good for assistive technologies, and search engine optimization. Please checkout this [link](https://www.w3schools.com/html/html5_semantic_elements.asp) about semantic html.
- [ ] Please also make sure you insert the proper background images as in the design. 
- [ ] Please make sure this section has the margins (top, bottom, left, right) defined in the design
- [ ] Please use the fonts and colors (background and font) defined for the template

     ## Your implementation ##    
![grace](https://user-images.githubusercontent.com/65852621/133791003-e778ff60-856a-45d2-b781-1359a644ee0e.png)

 ## The design ##     
![thedesign](https://user-images.githubusercontent.com/65852621/133791111-63bf54b4-4238-48f4-9277-d2ef5f20da87.png)